### DISCUSSION TITLE
Station location CITY requirement

### DISCUSSION THREAD

**Discussion by KB2YSI** [magicbug/Cloudlog#1538] [Q&A]
Is there a reason that the CITY field in a station location is required?

I frequently operate from many locations, satellite rover, and hardly know what county I am in much less than the town/city.  Having to enter that for each location is going to be either completely incorrect or a PITA to obtain the correct name.

**Comment by magicbug** [comment]
I'm happy to make this an unrequired field seems a reasonable suggestion :)

**Reply by KB2YSI** [reply]
Thank you for the reply Peter! I did not know if it was required for something else or not.

**Reply by phl0** [reply]
I was just about to need this as well and supplied a patch/PR. See https://github.com/magicbug/Cloudlog/pull/1572.

**Reply by phl0** [reply]
PR is merged. Please test and report.
